odoo/orm/models.py

```diff
@@ -6089,6 +6089,11 @@ def _get_redirect_suggested_company(self):
             return (self.company_ids & self.env.user.company_ids)[:1]
         return False
 
+    def _can_elevate_access(self, field: Field, access_token: str | None = None) -> bool:
+        """Check if extra access is allowed. It could be overridden
```

Comment:
The comment might be more specific than that.

"for the given field, in the context of downloading the field as a file or displaying it as an image."